serverless/knative-client.adoc

```diff
@@ -0,0 +1,40 @@
+[id="knative-client"]
+= Using Knative Client
+include::modules/common-attributes.adoc[]
+:context: knative-client
+
+toc::[]
+
+Knative Client (`kn`) is the Knative command line interface (CLI). The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}. With `kn`, you can create applications and manage {product-title} projects from the terminal. 
+
+Knative client does not have its own log in mechanism. To log in to the cluster you must install the `oc` CLI and use the `oc` login. Installation options for the CLI vary depending on your operating system.
+
+
+// Installation
+include::modules/cli-install.adoc[leveloffset=+1]
+include::modules/serverless-installing-cli-linux.adoc[leveloffset=+2]
+// RPM install procedure commented out
+// include::modules/serverless-installing-cli-linux-rpm.adoc[leveloffset=+2]
+include::modules/serverless-installing-cli-macosx.adoc[leveloffset=+2]
+include::modules/serverless-installing-cli-windows.adoc[leveloffset=+2]
+
+// Log in
+include::modules/cli-logging-in.adoc[leveloffset=+1]
+
+
+// Access workflows after installation
+The following workflows provide exaamples of how to use `kn`.
```

Comment:
exaamples -> examples

Reply:
But honestly I'd remove this sentence altogether. It gets lost once the doc is rendered and looks like it's actually a part of the "Logging in to the CLI" module.